test/v2/fixtures/foreignProperty/RetrieveForeignPropertyAdjustmentsFixtures.scala

```diff
@@ -58,4 +71,141 @@ object RetrieveForeignPropertyAdjustmentsFixtures {
   val foreignPropertyFhlEeaMinimalRetrieveForeignPropertyAdjustmentResponseModel =
     RetrieveForeignPropertyAdjustmentsResponse(foreignPropertyFhlEeaMetaDataModel, BsasDetail(None, None))
 
+  val hateoasResponseForForeignPropertyAdjustments: (String, String) => String = (nino: String, bsasId: String) =>
+    s"""
+       |{
+       |   "metadata": {
+       |      "typeOfBusiness": "foreign-property",
+       |      "accountingPeriod": {
+       |         "startDate": "2020-10-11",
+       |         "endDate": "2020-01-01"
+       |      },
+       |      "taxYear": "2019-20",
+       |      "requestedDateTime": "2020-10-14T11:33:27Z",
+       |      "bsasId": "717f3a7a-db8e-11e9-8a34-2a2ae2dbcce5",
+       |      "summaryStatus": "valid",
+       |      "adjustedSummary": true
+       |   },
+       |   "adjustments": {
+       |      "incomes": {
+       |         "rentIncome": 100.49,
+       |         "premiumsOfLeaseGrant": 100.49,
+       |         "otherPropertyIncome": 100.49
+       |      },
+       |      "expenses": {
+       |         "premisesRunningCosts": 100.49,
+       |         "repairsAndMaintenance": 100.49,
+       |         "financialCosts": 100.49,
+       |         "professionalFees": 100.49,
+       |         "travelCosts": 100.49,
+       |         "costOfServices": 100.49,
+       |         "residentialFinancialCost": 100.49,
+       |         "other": 100.49,
+       |         "consolidatedExpenses": 100.49
+       |      }
+       |   },
+       |	"links": [{
+       |		"href": "/individuals/self-assessment/adjustable-summary/$nino/foreign-property/$bsasId?adjustedStatus=true",
+       |		"method": "GET",
+       |		"rel": "retrieve-adjustable-summary"
+       |	}, {
+       |		"href": "/individuals/self-assessment/adjustable-summary/$nino/foreign-property/$bsasId/adjust",
+       |		"method": "GET",
+       |		"rel": "self"
+       |	}]
+       |}
+       |""".stripMargin
+
+  val hateoasResponseForeignPropertyAdjustments: (String, String) => String = (nino: String, bsasId: String) =>
+    s"""
+       |{
```

Comment:
why is there duplicate responses?

Reply:
the duplicate response was to fix the issue I was having in the tests, the integration spec and the unit tests were getting slightly different responses and I wasn't sure what else to do tbh

Reply:
so for example, one had a value for foreignTaxTakenOff, one didn't

it was easier to just make another response

Reply:
Hmm it just feels wrong having randomly a bunch of duplicates

Reply:
I mean, they're not exactly duplicates, tbf. they are different

Reply:
Whatever gets them going i guess